src/types/dialog.ts

```diff
@@ -24,11 +28,15 @@ export type DialogHeaderProps = {
 export type DialogBodyProps = {
     children: ReactNode;
     maxHeight?: number | string;
+    padding?: DialogSize;
     'data-test-id'?: string;
 };
 
 export type DialogFooterProps = {
-    buttons: ButtonProps[];
+    children?: ReactNode;
+    actionButtons: [ButtonProps] | [ButtonProps, ButtonProps];
```

Comment:
What if user in some case wants to have more that 2 actions buttons, he will not be able to specify those. I know that scenario is less likely to happen, but still. I would do simply:
```suggestion
    actionButtons: ButtonProps[]
```